django/db/models/aggregates.py

```diff
@@ -14,9 +14,9 @@ class Aggregate(Func):
     contains_aggregate = True
     name = None
 
-    def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize=False):
+    def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize=False, for_save=False):
         assert len(self.source_expressions) == 1
-        c = super(Aggregate, self).resolve_expression(query, allow_joins, reuse, summarize)
+        c = super(Aggregate, self).resolve_expression(query, allow_joins, reuse, summarize, False)
```

Comment:
That's a test case I missed ;) I'll check this. But now that you mention it, if this does work, I think the ``for_save`` would need to be passed to Min, and Max since they return the expression value.